src/Apps/Legion/Hooks/useDefaultNavigation.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useCallback } from 'react';
+import { useWizardNavigationContext } from '../Contexts/WizardNavigationContext/WizardNavigationContext';
+import {
+    IWizardAction,
+    WizardNavigationContextActionType,
+    WizardStepNumber
+} from '../Contexts/WizardNavigationContext/WizardNavigationContext.types';
+
+export const useDefaultNavigation = (
```

Comment:
I'm not seeing where you use this. Where can I see this consumed?

Reply:
Should every step/tab call either useCustom or useDefault? that way they make sure the controls make sense for themselves?

Reply:
useCustom uses useDefault if no primary action is given, this is the kind of hook building I am experimenting with works. Data pusher step will probably use useDefault navigation.